src/views/CardStack/CardStackStyleInterpolator.js

```diff
@@ -20,6 +23,26 @@ import type { NavigationSceneRendererProps } from '../../TypeDefinition';
  */
 
 /**
+ * Format the sequence of the scenes
+ * @param scenes
+ * @param index
+ * @returns {NavigationScene}
+ */
+function formatScenes(scenes: Array<NavigationScene>, index: number) {
```

Comment:
nit: JSDoc says this returns `NavigationScene`, it looks like you are returning `Array<NavigationScene>`. I would also add the appropriate Flow return type.

Reply:
The intention of this function is to correct the order of scenes, because in reset, there will be the same index of the scene, or the emergence of scene order reversed

Reply:
Can you add the appropriate flow return type to it? :)